### ISSUE TITLE
output file

### ISSUE THREAD

**Issue by YajunAlice** [eclufsc/ophidian#89]
Hi, i tried your code, and  find that when i run the test, there just some information about the memory, and i just don't know what's and where's the output files. Can you please give some guide about it ? Thank you.

**Comment by csguth** [comment]
Hi @YajunAlice,

Could you please paste the commands you're entering as well as the output you're getting?

**Comment by renannetto** [comment]
Hi @YajunAlice, the run_tests binary only runs the unit tests of the library. It can be used to check if the library has been succesfully compiled and linked, but it won't write any output files. If you could also tell us what output files are you expecting we can also help you.

**Comment by YajunAlice** [comment]
Sorry that I reply you so lately. The link below is the command that I use and the results it shows. First I just want to confirm that whether the output is what you initially mean, because when I compile your codes I changed  some codes in order to make the codes compile successfully. Second, I wonder how to use your codes to get the placement file for a given unplaced benchmark, can you give me more detail instructions on how to use your codes? Thank you for your help very much.
https://user-images.githubusercontent.com/31892002/31052844-c2dc0578-a6c1-11e7-8f9e-1c8015089732.png

**Comment by renannetto** [comment]
Yes, the output you're showing means your code is working correctly, since all tests passed.

About how to use our code, we provide a DesignBuilder class with two implementations: ICCAD2015ContestDesignBuilder (which builds a design from the ICCAD contest 2015 benchmarks files) and ICCAD2017ContestDesignBuilder (which builds a design from the ICCAD contest 2017 benchmarks files). 

Once you build a Design (from the DesignBuilder), you can access its netlist, placement, floorplan, etc. Then, you can implement your algorithms with this information. If you want to save the placement result in a file (like a DEF file), you will need to implement this functionality yourself, because we don't provide DEF writing functions yet.

**Comment by YajunAlice** [comment]
Got it, thank you very much!

**Comment by YajunAlice** [comment]
Hi, I want to confirm another question from you. Since I want to obtain some placement results  using some algorithms. From my understanding,  your project just gives us the whole structure that how to leverage the input files to construct a whole placed system, as for the relative placement algorithms we have to implement by ourselves, isn't it? So i can just write relative algorithms to obtain what i want by combine your codes? 
Since I have just joined this field recently, and do not very understand clearly. Thank you for helping me to understand it.

**Comment by renannetto** [comment]
Yes, our code provides the basic infrastructure (such as parsers and basic data structures) so that you can write your algorithms using it.
Feel free to ask if you have any more questions. You can also contact me by email (renan.o.netto@gmail.com) if you wish.

**Comment by tarberd** [comment]
This may be healpfull for you, [https://github.com/eclufsc/ophidian_project_template](url).

**Comment by YajunAlice** [comment]
Thank you very much. I will contact you by email if I have any questions.
